androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/TopControls.kt

```diff
@@ -0,0 +1,49 @@
+package com.profusion.androidenhancedvideoplayer.components.playerOverlay
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import com.profusion.androidenhancedvideoplayer.styling.Dimensions
+
+@Composable
+fun TopControls(
+    title: String? = null,
+    modifier: Modifier = Modifier
+) {
+    Box(
```

Comment:
Do we need the `Box` if we don't have content inside of it?

Reply:
Nope, but add it considering that it will be replaced in the future by a `Row` or `Column` with more children. e.g:

![image](https://github.com/profusion/android-enhanced-video-player/assets/35314270/ea69b8c8-b8b8-4018-a22c-71266d3d84ca)